api/v1/controllers/userTokens.js

```diff
@@ -18,25 +18,35 @@ const u = require('../helpers/verbs/utils');
 const httpStatus = cnstnts.httpStatus;
 const authUtils = require('../helpers/authUtils');
 
-function whereClauseForUser(user) {
+/**
+ * where clause to get user tokens
+ * @param  {String} userNameOrId - User name or id
+ * @returns {Object} - where clause
+ */
+function whereClauseForUser(userNameOrId) {
   const whr = {};
-  whr[cnstnts.SEQ_OR] = [
-    {
-      'user.name': {},
-    },
-    {
-      'user.id': {},
-    },
-  ];
-  whr[cnstnts.SEQ_OR][0]['user.name'][cnstnts.SEQ_LIKE] = user;
-  whr[cnstnts.SEQ_OR][1]['user.id'][cnstnts.SEQ_LIKE] = user;
+  if (u.looksLikeId(userNameOrId)) {
+    // need to use '$table.field$' for association fields
+    whr.where = { '$User.id$': {} };
+    whr.where['$User.id$'] = userNameOrId;
+  } else {
```

Comment:
aren't the where clauses defined something like this ?
where : {
    User: {
      id: "id" 
     }
}